examples/notebooks/GalSimTutorialCustomCatalog.ipynb

```diff
@@ -0,0 +1,772 @@
+    "\n",
+    "I recommend copying the `get_galacticAv` method below into any class that you find.  The getters for E(B-V) are a little buggy.  That should be fixed in the next release."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from lsst.sims.GalSimInterface import GalSimStars, SNRdocumentPSF\n",
+    "from lsst.sims.photUtils import PhotometricParameters\n",
+    "from lsst.obs.lsstSim import LsstSimMapper\n",
+    "\n",
+    "class PointSourceGalSimCatalog(GalSimStars):\n",
+    "\n",
+    "    photParams = PhotometricParameters()\n",
+    "    PSF = SNRdocumentPSF()\n",
+    "    \n",
+    "    sedDir = os.path.join(os.getcwd(),'testSEDs')\n",
+    "    bandpassDir = os.path.join(os.getcwd(),'testBandpasses')\n",
+    "    bandpassNames = ['y']\n",
+    "    bandpassRoot = 'myCustomBandpass_'\n",
+    "    componentList = []\n",
+    "    camera = LsstSimMapper().camera\n",
+    "\n",
+    "    def get_sedFilepath(self):\n",
+    "        # this method would be more complicated if there were sub-directories\n",
+    "        # of sedDir that were not stored in the database\n",
+    "        return self.column_by_name('sedFilename')\n",
+    "    \n",
+    "    def get_galacticAv(self):\n",
+    "        ra = self.column_by_name('raJ2000')\n",
+    "        if len(ra)==0:\n",
+    "            return []\n",
+    "\n",
+    "        ebv = self.column_by_name('EBV')\n",
+    "        return 3.1*ebv"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to actually create a catalog, we need an instantiation of the `ObservationMetaData` class to tell the `GalSimCatalog` where the telescope is pointing.  `boundType` and `boundLength` will determine the field of view queried from the `fileDBObject`.  Because the `GalSimCatalog` is associating objects with detectors, you just need to make sure that the returned field of view is larger than the field of view of the camera.  Do not worry about making the field of view exactly the same size as the camera's field of view."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "from lsst.sims.utils import ObservationMetaData\n",
+    "start = time.clock()\n",
+    "obs = ObservationMetaData(unrefractedRA=52.0, unrefractedDec=-26.0,\n",
+    "                          boundType='circle', boundLength=2.0,\n",
+    "                          mjd=52000.0, rotSkyPos=37.0)\n",
+    "\n",
+    "cat = PointSourceGalSimCatalog(starDB, obs_metadata=obs)\n",
+    "cat.write_catalog('testOutputCatalogs/point_source_test_catalog.txt')\n",
+    "cat.write_images(nameRoot='testImages/pointSourceImage')\n",
+    "print time.clock()-start"
+   ]
+  },
+  {
```

Comment:
Am I supposed to get this error:
    /Users/Bryce/lsst/DarwinX86/anaconda/master-g68783b1848/lib/python2.7/site-packages/sqlalchemy/sql/type_api.py:322: SAWarning: Dialect sqlite+pysqlite does *not* support Decimal objects natively, and SQLAlchemy must convert from floating point - rounding errors and other issues may occur. Please consider storing Decimal numbers as strings or integers on this platform for lossless storage.
  d[coltype] = rp = d['impl'].result_processor(dialect, coltype)